caleb-lab/sll.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

Need to comment before each method on its Big Oh notation.